docs/internals.rst

```diff
@@ -174,13 +174,6 @@ function be called.  For example, if you were writing a middleware which cached
 responses for certain methods your middleware would likely not call the
 ``make_request`` method, but instead get the response from some local cache.
 
-By default, Web3 will use the ``web3.middleware.pythonic_middleware``.  This
-middleware performs the following translations for requests and responses.
-
-* Numeric request parameters will be converted to their hexadecimal representation
-* Numeric responses will be converted from their hexadecimal representations to
```

Comment:
I could be mistaken but I don't actually see any good detail on formatters in the docs. Since this moved to the formatters we should definitely add a section explaining what they do and how they work, etc. I would say that's beyond the scope of this PR but I can create an issue to track it.